r_standards.md

```diff
@@ -0,0 +1,50 @@
+# R Style Guidelines
+The ACF Data Surge team's R Style Guidelines are intended to make our R code consistent and easy to read across projects. Our guidelines align with the [The tidyverse style guide](https://style.tidyverse.org/) and we adopt the [Google R Style Guide](https://google.github.io/styleguide/Rguide.html) approach of listing the ways that our guidelines differ from the tidyverse guidelines. 
+
+Our guidelines are unique in that we include a section at the end detailing preferred packages and package management standards.
+
+## Functions
+
+### Returning Values
+
+- Use return() explicitly for returning values from a function
+```
+my_mean <- function(x) {
+    x_mean <- mean(x)
+    return(x_mean)
+}
+```
+
+## Pipes
+
+### Assignment
+- Avoid right-handed assignment
```

Comment:
FWIW, I prefer using "do" vs. "don't" framing where possible. So in this case it'd be "Use left-handed assignment"